cleanrl/dqn.py

```diff
@@ -201,9 +201,9 @@ def linear_schedule(start_e: float, end_e: float, duration: int, t: int):
             loss.backward()
             optimizer.step()
 
-            # update the target network
-            if global_step % args.target_network_frequency == 0:
-                target_network.load_state_dict(q_network.state_dict())
+        # update the target network
+        if global_step % args.target_network_frequency == 0:
+            target_network.load_state_dict(q_network.state_dict())
```

Comment:
Maybe change it to 

```
        # update the target network
        if global_step > args.learning_starts and global_step % args.target_network_frequency == 0:
            target_network.load_state_dict(q_network.state_dict())
```

Reply:
What about something like this instead:

```python
if global_step > learning_starts :
    if global_step % train_frequency == 0:
        ...

    if global_step % target_network_frequency == 0:
        ...
```

I find it more readable.

Reply:
That would work great.